passes/source2il.nim

```diff
@@ -258,6 +266,12 @@ proc genAsgn(c; a: Node|NodeSeq, b: NodeSeq, typ: SemType, bu) =
       bu.add a
       genUse(b, bu)
 
+proc genDrop(a: Node|NodeSeq, typ: SemType, bu) =
```

Comment:
```suggestion
proc genDrop(a: NodeSeq, typ: SemType, bu) =
```
Alternatively, `genUse` could use `Node|NodeSeq` for its parameter.

Reply:
I went with the alternative.